doc_user/UserManual.adoc

```diff
@@ -828,6 +829,409 @@ image:image251.png[success]
+
+===== 4.5.2.1 Create a New ward Inventory ([.underline]##N##ew)
+
+The *New Inventory* function is used to create a new inventory for a ward. Click on *[.underline]##N##ew* button in the *_Manage Ward Inventory_* window to access the *_New Inventory_* window.
+
+image:image263.png[New ward Inventory]
+
+After the selection of the ward (FEMALE WARD in this example), the *_New Inventory_* window is shown, allowing for the management of the FEMALE WARD inventory:
+
+image:image264.png[New ward Inventory]
+
+The *_New Inventory_* window is composed of three areas: the *Inventory panel* at the top, the *Inventory row table* in the center and the *Buttons panel* on the bottom.
+
+The data shown in the *Upper panel* area are:
+
+* *Select ward*: the ward that is associated with this inventory.
+* *Date*: the date of the inventory.
++
+NOTE: This affects the _validation_ step, when the system will check all further charges and/or discharges recorded after this date and so potentially affecting the theoretical and real quantities in the current inventory.
+* *Inventory's reference*: the reference associated to the inventory.
+* *Add a medical*: this field is used to add inventory row for a medical.
+* *Add medicals by*: This button is used to add inventory row for a set of medicals.
+
+The data shown in the *Center panel* area represents an inventory row. The data are:
+
+* *CODE*: the code of the pharmaceutical.
+* *MEDICAL*: the description of the pharmaceutical.
+* *NEW LOT*: indicates if the inventory row introduces a new lot (never recorded before) for the system.
+* *LOT's CODE.*: the lot code of the pharmaceutical.
+* *DUE DATE*: the due date of the lot.
+* *THEORETICAL*: the theoretic quantity of the lot of the pharmaceutical.
+* *REAL QUANTITY*: the real(physical) quantity of the lot of the pharmaceutical.
+* *DIFFERENCE*: the difference between real and theoretical quantities, if any.
+* *UNIT COST*: the unit cost of the pharmaceutical.
+* *TOTAL COST*: the total cost or real value of the lot.
+
+The *Buttons panel* is used to perform some action:
+
+* *[.underline]##S##ave*: this button is used to save the inventory.
+* *[.underline]##L##ot*: this button is used to link a new lot to a new specific product (the inventory row that column *New Lot* as value *N*). To perform the operation, first highlighted the product and click this button.
+
+NOTE: This button is enabled only for the inventory row where the column *New Lot* has value the value *N*, otherwise the button is disabled.
+
+* *[.underline]##D##elete*: this button is used to remove one or a set of inventory rows.
+* *Clean [.underline]##T##able*: this button is used to remove all the inventory rows.
+* *[.underline]##V##alidate*: this button is used to validate the inventory.
+* *[.underline]##C##onfirm*: this button is used to confirm the inventory.
+
+NOTE: The Confirm button is enabled only if the inventory is validated, otherwise the button is disabled. See 
```

Comment:
Nothing seems to follow the "See" at the end of the line

Reply:
Yeah, fixed, thanks!